sql-plugin/src/main/scala/com/nvidia/spark/rapids/SparkShims.scala

```diff
@@ -115,6 +115,7 @@ trait SparkShims {
      exportColumnRdd: Boolean): GpuColumnarToRowExecParent
   def getExecs: Map[Class[_ <: SparkPlan], ExecRule[_ <: SparkPlan]]
   def getScans: Map[Class[_ <: Scan], ScanRule[_ <: Scan]]
+  def getFileFormats: Map[FileFormatType, Map[FileFormatOp, FileFormatChecks]] = Map()
```

Comment:
It does not require a default implementation.
```suggestion
  def getFileFormats: Map[FileFormatType, Map[FileFormatOp, FileFormatChecks]]
```

Reply:
It's needed by other Spark shims except the Shim33x, Shim33x overwrited it.